Simplenote/NSProgressIndicator+Simplenote.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+extension NSProgressIndicator {
+    static func addProgressIndicator(to view: NSView) -> NSProgressIndicator {
+        let progressIndicator = NSProgressIndicator.init(frame: NSRect(x: view.frame.minX, y: view.frame.midY, width: 20, height: 20))
```

Comment:
```suggestion
        let progressIndicator = NSProgressIndicator(frame: NSRect(x: view.frame.minX, y: view.frame.midY, width: 20, height: 20))
```

Reply:
Since you're using autolayout... can we try just running `NSProgressIndicator()` instead?

Reply:
Plus let's also add `progressIndicator.controlSize = .small`, looks way cooler!